packages/popover/src/types.ts

```diff
@@ -115,4 +107,25 @@ export interface PopoverProps {
    * default: false
    */
   adjustOnMutation?: boolean;
-}
+} & (
+  | {
+      /**
+       * Specifies that the popover content will appear portaled to the end of the DOM,
+       * rather than in the DOM tree.
+       *
+       * default: `true`
+       */
+      usePortal?: true;
```

Comment:
Do we need to add the TSDoc annotations to the other instance of this prop below?

Reply:
I actually don't know where the TSDoc annotations are used. To me it feels a bit redundant with the existing documentation of the the component.